src/main/java/org/opensearch/security/tools/democonfig/Installer.java

```diff
@@ -356,11 +383,19 @@ static void finishScriptExecution() {
                 System.out.println("### To use the Security Plugin ConfigurationGUI");
             }
 
-            System.out.println("### To access your secured cluster open https://<hostname>:<HTTP port> and log in with admin/admin.");
+            System.out.println(
+                "### To access your secured cluster open https://<hostname>:<HTTP port> and log in with admin/"
+                    + SecuritySettingsConfigurer.ADMIN_PASSWORD
+                    + "."
+            );
             System.out.println("### (Ignore the SSL certificate warning because we installed self-signed demo certificates)");
 
         } catch (Exception e) {
             System.out.println(e.getMessage());
         }
     }
+
+    public static void resetInstance() {
```

Comment:
If this isn't used anywhere can it be removed?

Reply:
It is used in tests